bittensor/metagraph.py

```diff
@@ -18,12 +18,49 @@
 
 from bittensor.exceptions.handlers import rollbar
 
+MAX_INT_WEIGHT = 4294967295 # Max weight value on chain.
+
```

Comment:
If someone changes this, would it break things on the chain?

Reply:
It would destroy the universe.